src/status.h

```diff
@@ -109,11 +109,11 @@ struct StatusPrinter : Status {
         rate_ = times_.size() / ((times_.back() - times_.front()) / 1e3);
     }
 
-  private:
-    double rate_;
-    const size_t N;
-    queue<double> times_;
-    int last_update_;
+    private:
```

Comment:
This is pretty subjective.

Personally I prefer it the old way :/

Reply:
Agreed, totally subjective.